package/Linux.mk

```diff
@@ -98,7 +98,9 @@ install:
 	$(V1) $(MKDIR) -p $(DESTDIR)$(datadir)
 	$(V1) $(MKDIR) -p $(DESTDIR)$(datadir)/applications
 	$(V1) $(MKDIR) -p $(DESTDIR)$(datadir)/pixmaps
-	$(V1) $(MKDIR) -p $(DESTDIR)$(udevdir)
+# Following line creates udev directory - however udevdir is not previously defined, nor is anything
+# installed to this location in the rest of this recipe
+#	$(V1) $(MKDIR)  $(DESTDIR)$(udevdir)
```

Comment:
I wondered about this at the time I made the install rule but debian package install udev rules separately so I left it out, making it optional would be best